src/Uno.UI/UI/Xaml/Media/Animation/ColorAnimation.cs

```diff
@@ -50,6 +50,18 @@ public bool EnableDependentAnimation
 			}
 		}
 
+		public bool AutoReverse
+		{
+			get
+			{
+				return (bool)this.GetValue(AutoReverseProperty);
+			}
+			set
+			{
+				this.SetValue(AutoReverseProperty, value);
+			}
+		}
```

Comment:
`AutoReverse` is not a property on `ColorAnimation` on UWP/WinUI: it's a property on `TimeLine`.

https://github.dev/Hudney/uno/blob/cc811c565cc9a74459c6fd63f7f47273fcdf5d59/src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Media.Animation/Timeline.cs#L28-L40